app/models/site_statistic.rb

```diff
@@ -120,7 +120,31 @@ def self.observations_stats(at_time = Time.now)
               gte: at_time - 7.days,
               lte: at_time
             }
-          },
+          }
+        },
+        {
+          exists: { field: "community_taxon_id" }
+        }
+      ]
+    ).total_entries
+    community_identified_to_genus = Observation.elastic_search(
```

Comment:
This count doesn't quite match up to the original, and it looks like we don't have the data in Elasticsearch to get the same count. This query returns the count of observations that have a community_taxon_id and where the taxon is rank genus or below - not where the `community_taxon` is rank genus or below as observation.community_taxon isn't necessarily the same as observation.taxon. We don't store the community_taxon rank in elasticsearch. This ends up being few percent higher than the original value.

I wonder if it's worth keeping the old long-to-calculate stats, or maybe this is even a stat we could get rid of entirely. Alternatively we could do something more complicated like fetching all the unique community_taxon_ids from ES, then querying the DB to figure out which ones are genera or below

Reply:
Community taxon has been fixed